### PR TITLE
Add commands/web-searches/search-script-command-marketplace

### PR DESCRIPTION
**Pull request by Pondorasti**
## Description

Created new utility script that searches for script commands in the Unofficial Marketplace, [scriptcommands.com](scriptcommands.com).

## Type of change

- [x] New script command

## Screenshot

https://user-images.githubusercontent.com/32957606/125560446-3d20b2ba-8716-42eb-ae0b-13859c132936.mov

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Pondorasti** [comment]
@unnamedd 🕵️‍♂️ 👀

File: commands/developer-utils/search-script-command-marketplace.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search Script Command
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon images/marketplace-logo.png
+# @raycast.argument1 { "type": "text", "placeholder": "query"}
+# @raycast.packageName Marketplace Search
+# @raycast.description Search for Script Commands in the Unofficial Marketplace, scriptcommands.com
```
Comment (unnamedd) [inline commands/developer-utils/search-script-command-marketplace.sh:12]:
```suggestion
# @raycast.description Search for Script Commands in the [Unofficial Raycast Script Commands Marketplace](https://scriptcommands.com).
```

File: commands/developer-utils/search-script-command-marketplace.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search Script Command
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon images/marketplace-logo.png
+# @raycast.argument1 { "type": "text", "placeholder": "query"}
+# @raycast.packageName Marketplace Search
```
Comment (unnamedd) [inline commands/developer-utils/search-script-command-marketplace.sh:11]:
```suggestion
# @raycast.packageName Web Searches
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @Pondorasti,

thanks for the website you've created for the Script Commands and also for this Script Command as well. It's funny to think that you will find this Script Command in your website to search for Script Commands... haha like chicken and egg problem, a paradox! :P

I've made some suggestion but the last one I need to do here: this Script Command will fit better in the web-searches folder instead of developer-utils.

Thanks one more time!

File: commands/developer-utils/search-script-command-marketplace.sh (hunk 2 above)
Comment (Pondorasti) [reply commands/developer-utils/search-script-command-marketplace.sh]: Fixed, but wouldn't it make more sense to just use `Marketplace` as `packageName`?

Comment (unnamedd) [reply commands/developer-utils/search-script-command-marketplace.sh]: Following the standard already created, I'd say: no. If you check for example the Script Commands for some stores we have inside of the web-searches folder, they also have in the `packageName`: "Web Searches".

Comment (unnamedd) [reply commands/developer-utils/search-script-command-marketplace.sh]: Of course, unless we decide to give an upgrade on them and change the packageName for the Script Commands related to Marketplaces.

Comment (Pondorasti) [reply commands/developer-utils/search-script-command-marketplace.sh]: Yup, that makes sense. The beauty of it is that I can customize it however I like on my device, hehe.

**Review by unnamedd — APPROVED** [review]
Looks good to me!

**Review by mattisssa — APPROVED** [review]
Thanks for the PR! Looks good to me 👍

**Pull request merged**